OWNERS

```diff
@@ -0,0 +1,8 @@
+approvers:
+- aweiteka
+- jewzaam
+- mwoodson
+reviewers:
+- aweiteka
+- jewzaam
+- mwoodson
```

Comment:
```suggestion
- mwoodson
- lisa
```

I'd like to remain a reviewer on this one.

Reply:
chat on slack, you can still lgtm from what I've observed, this is just approval